src/core/mac/sub_mac.hpp

```diff
@@ -667,7 +667,7 @@ class SubMac : public InstanceLocator, private NonCopyable
     bool mIsCslSampling : 1;  // Indicates that the radio is receiving in CSL state for platforms not supporting delayed
                               // reception.
     uint16_t    mCslPeerShort;      // The CSL peer short address.
-    TimeMicro   mCslSampleTime;     // The CSL sample time of the current period.
+    TimeMicro   mCslSampleTime;     // The CSL sample time of the current period referred to the local radio clock.
```

Comment:
```suggestion
    TimeMicro   mCslSampleTime;     // The CSL sample time of the current period referenced to the local radio clock.
```

Reply:
Or "relative to" is even more clear (but in that case the same wording should be used in other places as well)

Reply:
You are right "relative to" is a common way to express what I mean here (see PTP spec for example). Fixed everywhere.